tests/boot/boot_from_pxe.pm

```diff
@@ -21,27 +21,58 @@ use testapi;
 use bootloader_setup qw(bootmenu_default_params specific_bootmenu_params);
 use registration 'registration_bootloader_cmdline';
 use utils 'type_string_slow';
+use backend::ipmi 'ipmitool';
+
+# Variables are present in vars.json
+# IPMI_HOSTNAME, IPMI_USER, IPMI_PASSWORD
+my $ipmi_host = {
+    hostname => get_required_var('IPMI_HOSTNAME'),
+    username => get_required_var('IPMI_USER'),
+    userpass => get_required_var('IPMI_PASSWORD'),
+};
+
+sub reactivate_sol {
+    diag("IPMI host name is $ipmi_host->{hostname}");
+    diag("IPMI Activate SOL");
+    my $rc = system("ipmitool -I lanplus -H $ipmi_host->{hostname} -U $ipmi_host->{username} -P $ipmi_host->{userpass} -vv sol activate");
+    # EXIT_FAILURE of ipmitool
+    if (($rc != 0) || ($ipmi_host->{hostname} =~ m/sp.scooter.qa.suse.de/)) {
```

Comment:
Why is sp.scooter.qa.suse.de hardcoded here ?

Reply:
Empirically said, there is a problem to connect to SOL.
http://eris.suse.cz/tests/14543

Mostly it happened in scooter, but I have seen it on others as well, but way fewer occurrences.

Reply:
But this could be done for every machine ?